### DISCUSSION TITLE
baize/developer/notebooks/notebook-with-ssh/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#4322] [Announcements]
# baize/developer/notebooks/notebook-with-ssh/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/baize/developer/notebooks/notebook-with-ssh/?h=ssh

**Comment by panpan0000** [comment]
添加 SSH 公钥到个人中心¶ 是可选的

**Comment by ModetaNiu** [comment]
fixed,  see https://github.com/DaoCloud/DaoCloud-docs/pull/4546
